pkg/controller/storagecluster/reconcile.go

```diff
@@ -84,9 +84,8 @@ func (r *ReconcileStorageCluster) Reconcile(request reconcile.Request) (reconcil
 	reqLogger := r.reqLogger.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 
 	// Fetch the StorageCluster instance
-	instance := &ocsv1.StorageCluster{}
-	err := r.client.Get(context.TODO(), request.NamespacedName, instance)
-	if err != nil {
+	sc := &ocsv1.StorageCluster{}
```

Comment:
This variable rename probably belongs in another commit.

Reply:
This is not a var rename, this is a totally new function that took the name of the old one (`reconcile`). 
The code of the old function was renamed `reconcilePhases` and there I did not touch the name of the var (it is still instance). It just that git diff is a little bit awkward in these situations. 
As such I don't see a problem with this line